### DISCUSSION TITLE
Is there a way to redeem a code using steamworks-rs?

### DISCUSSION THREAD

**Discussion by Jev1337** [Thinkofname/steamworks-rs#26] [Q&A] [answered]
I'm kind of a beginner with rust lang and I want to try something as such:

Put a key into console and it tries to redeem it on my account for example.

**Comment by Thinkofname** [comment, marked as answer]
As far as I'm aware the steam API doesn't provide a way to do that and its restricted to being something only the steam website/app can do.
